spec/services/precompute_feed_service_spec.rb

```diff
@@ -20,6 +20,8 @@
     it 'does not raise an error even if it could not find any status' do
       account = Fabricate(:account)
       subject.call(account)
+
+      pending 'Implement check to not raise error'
```

Comment:
Could wrap the subject.call with a block and assert no error raised

Reply:
Yeah, I could add `expect(subject.call(account)).to_not raise_error`, but I'm not sure that actually tests what the case is talking about. Think it would somehow need to do a call where there is some query for a non-existant account

Reply:
Ah, I see -- sorry, I didn't look at the larger context.

Looks like it was added here - https://github.com/mastodon/mastodon/pull/4015/files - and maybe looking for a scenario where `pairs` in the PrecomputeFeedService was not as expected.

That file has changed a lot since that commit, so it's possible that this either isn't relevant anymore, or the possible error condition is elsewhere in code now.

Reply:
OK, I'll leave the pending for now, so someone can try and figure it out later